Reorder GuiComponent

Goldorion opened the pull request:
This thing https://github.com/MCreator/MCreator/pull/3477#pullrequestreview-1264752710

KlemenDEV reviewed (commented):
![image](https://user-images.githubusercontent.com/16374228/214141061-1f468aae-5867-4a9b-a74d-0e1ad6054f77.png)

In my tests, textfield renders above the label

Goldorion commented:
> ![image](https://user-images.githubusercontent.com/16374228/214141061-1f468aae-5867-4a9b-a74d-0e1ad6054f77.png)
> 
> In my tests, textfield renders above the label

I just changed what I changed in the original PR (so buttons and after)

KlemenDEV commented:
This is why I wanted this to be a separate PR so we also fix the order. I would also multiply all weights by 10 so there is space in between for other components we may add in the future so we don't need to re-order them every time

KlemenDEV commented:
This is needed first for #3483 and #3477 to be merged

Goldorion commented:
> This is needed first for #3483 and #3477 to be merged

I changed everything in the other PR, so this one can be merged before the other... if I knew this before 😞

KlemenDEV commented:
Not sure I understood what you mean. What I meant is this PR changes need to be finished and merged before other two are merged or we may have to re-order again for those PRs depending on when either of them is merged

Goldorion commented:
> Not sure I understood what you mean. What I meant is this PR changes need to be finished and merged before other two are merged or we may have to re-order again for those PRs depending on when either of them is merged

Oh sorry, I read the opposite thing 😅😓

Goldorion commented:
> This is why I wanted this to be a separate PR so we also fix the order. I would also multiply all weights by 10 so there is space in between for other components we may add in the future so we don't need to re-order them every time

Everything is re-ordered, and weights are also changed, so they have a gap between them. I also changed the order of the GUI editor, so they are also ordered depending on their weight.
